CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 * add a preprocessor to enrich by systems env variables
 * add option to define rules inline in pipeline config under processor configs `generic_rules` or `specific_rules`
+* add option to `field_manager` to ignore missing source fields to suppress warnings and failure tags
+* add ignore_missing_source_fields behavior to `calculator`, `concatenator`, `dissector`, `grokker`, `ip_informer`, `selective_extractor`
```

Comment:
why are these two separate bullet points? isn't it the same change, like you could just list the `field_manager` first in the second bullet point processor list.

Reply:
no... we could implement it only for the field_manager. that is the base implementation. but after I was done with that I thought  about it and it came to my mind, that this could do very stupid things in childs of the FieldManagerRule :-D
so it is a second implementation, that no one asked for.